ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2374,3 +2374,44 @@ def test_torch_sgn(
         input=input[0],
         out=None,
     )
+
+
+@handle_frontend_test(
+    fn_tree="torch.nan_to_num",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+        min_value=-100,
+        max_value=100,
+        allow_nan=True,
+        allow_inf=True,
+    ),
+    nan=st.floats(min_value=0.0, max_value=100),
```

Comment:
```suggestion
    nan=st.floats(min_value=-100.0, max_value=100.0),
```

Let's also allow for negative values here

Reply:
hi @ZiadAmerr, modified the test cases in latest PR commit here to accept negative values.